cluster/juju/layers/kubernetes-worker/actions/registry

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+# Example usage :
+#
+# Registry creation
+#
+# juju run-action kubernetes-worker/0 registry domain=myregistry.internal \
+# htpasswd="$(cat htpass)" ingress=true \
```

Comment:
Do we really need to base64 encode it? we're passing around TLS certs which are carriage return sensitive over the relation wires and haven't base64 encoded them. It hasn't seemed to be an issue yet, we might be playing with fire here, but in my experience, base64 encoding the contents is an extra, unnecessary step.